lib/mongoose/index.js

```diff
@@ -39,6 +39,8 @@ const MongoosePlugin = {
 
         Promise.all(queue).then(() => {
             next();
+        }).catch((e) => {
+            console.log('[mongoose] ', e);
```

Comment:
No need to that extra space after ] unless we are directly concatenating e with it.